### ISSUE TITLE
Not working anymore :(

### ISSUE THREAD

**Issue by whitneyit** [css-modules/jspm-demo#4]
Cloning this repo and running `jspm install` yields the following error.

> (program):1 Uncaught SyntaxError: Unexpected token {
> localhost/:1 Uncaught (in promise) Uncaught SyntaxError: Unexpected token {
>      Evaluating http://localhost:3333/src/main.css
>      Error loading http://localhost:3333/src/main.js

**Comment by whitneyit** [comment]
It appears to be trying to parse the file as a js

**Comment by geelen** [comment]
Yeah damn. The new JSPM dropped yesterday and I haven't had a chance to fix this yet. Soon though!

You can try to roll back to JSPM `v0.15.x` to try it out. I know @superhighfives got this running yesterday, I think that was all he did.

**Comment by whitneyit** [comment]
Ok thanks mate. I'll give it a shot

**Comment by superhighfives** [comment]
Hey @whitneyit — I've pulled together a working example that covers CSS Modules, jspm-css-loader with plugins like PostCSS, and vanilla CSS.

You can see it here:
https://github.com/superhighfives/css-modules-react-jspm-example

It also uses the `v0.15.x` JSPM, which, for me, solved the issue you came across.

**Comment by whitneyit** [comment]
Awesome thanks man I'll check it out!
